NFe.AppTeste.NetCore/NFe.AppTeste.NetCore.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Aqui pode ficar 2.1 sem problemas pois a 2.2 está em end of life: https://dotnet.microsoft.com/download/dotnet-core